modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc

```diff
@@ -6,32 +6,41 @@
+====
+
+.Firmware compatibility for HP hardware with Redfish virtual media 
 [frame="topbot", options="header"]
+[cols="1,1,1"]
 |====
-|Hardware| Model | Management | Firmware versions
-| HP | 10th Generation | iLO5 | 2.63 or later
-
-.2+| Dell | 14th Generation | iDRAC 9 | v4.20.20.20 - v4.40.00.00 only
-
-| 13th Generation .2+| iDRAC 8 | v2.75.75.75 or later
+| Model | Management | Firmware versions
+| 10th Generation | iLO5 | 2.63 or later
 
 |====
 
 [NOTE]
 ====
-Red Hat does not test every combination of firmware, hardware, or other third-party components. For further information about third-party support, see link:https://access.redhat.com/third-party-software-support[Red Hat third-party support policy].
+For HP servers, Redfish virtual media is not supported on 9th generation systems running iLO4, because Ironic does not support iLO4 with virtual media.
+====
+
+.Firmware compatibility for Dell hardware with Redfish virtual media
+[frame="topbot", options="header"]
+|====
+| Model | Management | Firmware versions
 
-See the hardware documentation for the nodes or contact the hardware vendor for information about updating the firmware.
+| 15th Generation | iDRAC 9 | v5.10.00.00 - v5.10.50.00 only
+| 14th Generation | iDRAC 9 | v5.10.00.00 - v5.10.50.00 only
 
-For HP servers, Redfish virtual media is not supported on 9th generation systems running iLO4, because Ironic does not support iLO4 with virtual media.
+| 13th Generation .2+| iDRAC 8 | v2.75.75.75 or later
 
-For Dell servers, ensure the {product-title} cluster nodes have AutoAttach Enabled through the iDRAC console. The menu path is: *Configuration* -> *Virtual Media* -> *Attach Mode* -> *AutoAttach* . With iDRAC 9 firmware version `04.40.00.00`, the Virtual Console plug-in defaults to `eHTML5`, which causes problems with the *InsertVirtualMedia* workflow. Set the plug-in to `HTML5` to avoid this issue. The menu path is: *Configuration* -> *Virtual console* -> *Plug-in Type* -> *HTML5* .
-====
+|====
 
-[IMPORTANT]
+[NOTE]
 ====
-The installer will not initiate installation on a node if the node firmware is below the foregoing versions when installing with virtual media.
+
+For Dell servers, ensure the {product-title} cluster nodes have AutoAttach Enabled through the iDRAC console. The menu path is: *Configuration* -> *Virtual Media* -> *Attach Mode* -> *AutoAttach* . With iDRAC 9 firmware version `04.40.00.00`, the Virtual Console plug-in defaults to `eHTML5`, which causes problems with the *InsertVirtualMedia* workflow. Set the plug-in to `HTML5` to avoid this issue. The menu path is: *Configuration* -> *Virtual console* -> *Plug-in Type* -> *HTML5* .
```

Comment:
probably change to:
firmare version `04.40.00.00` and later
or:
starting from firmare version `04.40.00.00`